concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -1026,20 +1020,14 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_task_
         self.assertEqual(message_file_transfer_token.operation, FileTransferToken.Operation.upload)
 
         # STEP 3: Requestor receives force get task result download because Provider uploaded file.
-        with mock.patch(
-            'core.transfer_operations.request_upload_status',
-            side_effect=request_upload_status_true_mock
-        ) as request_upload_status_true_mock_function:
-            with freeze_time("2017-12-01 11:00:54"):
-                response_3 = self.client.post(
-                    reverse('core:receive'),
-                    data=self._create_requestor_auth_message(),
-                    content_type='application/octet-stream',
-                )
+        result_upload_finished(deserialized_task_to_compute.subtask_id)  # pylint: disable=no-value-for-parameter
 
-        request_upload_status_true_mock_function.assert_called_with(
-            deserialized_report_computed_task
-        )
+        with freeze_time("2017-12-01 11:00:54"):
+            response_3 = self.client.post(
+                reverse('core:receive'),
+                data=self._create_requestor_auth_message(),
+                content_type='application/octet-stream',
+            )
 
         self.assertEqual(response_3.status_code,  200)
         self._test_subtask_state(
```

Comment:
Good thing might be to change style in whole file if you touched many places there. Please do it as a separate commit

Reply:
:+1: done